static/app/components/events/interfaces/spans/scrollbarManager.tsx

```diff
@@ -90,14 +90,17 @@ export class Provider extends Component<Props, State> {
 
     const anchoredSpanHash = window.location.hash.split('#')[1];
 
+    // If we're on the issues page, this is an embedded span tree. I know, this is hacky and weird but idk how else to check for this
+    const isEmbeddedSpanTree = window.location.href.includes('issues');
```

Comment:
Add an `isEmbedded` prop to `traceView` and pass it down from `EmbeddedSpanTree`.